ext/intl/transliterator/transliterator_class.c

```diff
@@ -183,7 +183,7 @@ static zend_object *Transliterator_clone_obj( zval *object )
 				"Could not clone transliterator", 0 );
 
 			err_msg = intl_error_get_message( TRANSLITERATOR_ERROR_P( to_orig ) );
-			php_error_docref( NULL, E_ERROR, "%s", ZSTR_VAL(err_msg) );
+			zend_throw_error( zend_ce_error, "%s", ZSTR_VAL(err_msg) );
```

Comment:
There's a comment below about not destroying `tempz`, which is derived from `ret_val`, so I'm not entirely sure if either should be freed.